IdentityCore/src/cache/MSIDKeychainTokenCache.m

```diff
@@ -811,7 +811,9 @@ - (NSArray *)itemsWithKey:(MSIDCacheKey *)key
     {
         if (error)
         {
-            *error = MSIDCreateError(MSIDKeychainErrorDomain, status, @"Failed to get items from keychain.", nil, nil, nil, context.correlationId, nil, NO);
+            NSMutableDictionary *additionalUserInfo = [NSMutableDictionary new];
```

Comment:
Nit: Please consider using a non-mutable dictionary if the dict is created new each time

Reply:
Will update later at all places